src/diffusers/loaders.py

```diff
@@ -1168,6 +1168,8 @@ class LoraLoaderMixin:
     """
     text_encoder_name = TEXT_ENCODER_NAME
     unet_name = UNET_NAME
+    loras_loaded = 0
```

Comment:
would we do this maybe in a v2 - I'd prefer to keep the v1 version as simple as possible and store only __call__ arguments

Reply:
Yeah agreed.